pillarbox-demo/src/main/java/ch/srgssr/pillarbox/demo/ui/player/controls/PlayerBottomToolbar.kt

```diff
@@ -61,22 +63,25 @@ fun PlayerBottomToolbar(
     onShuffleClick: (() -> Unit)?,
     repeatMode: @Player.RepeatMode Int,
     onRepeatClick: (() -> Unit)?,
-    pictureInPictureEnabled: Boolean,
-    onPictureInPictureClick: (() -> Unit)?,
+    isPictureInPictureEnabled: Boolean,
+    isInPictureInPicture: Boolean,
+    onPictureInPictureClick: () -> Unit,
     fullScreenEnabled: Boolean,
     onFullscreenClick: (() -> Unit)?,
     onSettingsClick: () -> Unit,
 ) {
     Row(modifier = modifier) {
         CompositionLocalProvider(LocalContentColor provides Color.White) {
             ToggleableIconButton(
+                enabled = true,
                 checked = shuffleEnabled,
                 icon = if (shuffleEnabled) Icons.Default.ShuffleOn else Icons.Default.Shuffle,
                 contentDestination = stringResource(R.string.shuffle),
                 onCheckedChange = onShuffleClick,
             )
 
             ToggleableIconButton(
+                enabled = true,
```

Comment:
We could check if the command `Player.COMMAND_SET_REPEAT_MODE` is available ? Maybe a state object for him. I guess it is done in the other related pr?

Reply:
Yes, I'll submit a dedicated PR to include state objects provided by Media3. I wanted to focus on the PiP in this one.